doc/tutorial/staramr-tutorial.ipynb

```diff
@@ -370,35 +433,148 @@
+      "GCF_001931595.1  gyrA (D87Y)  ciprofloxacin I/R, nalidixic acid                                99.431             100.0              2637/2637                ref|NZ_CP016410.1|      1597907.0  1600543.0             Resistance\n",
+      "GCF_001931595.1  sul1         sulfisoxazole                                                    100.0              100.0              840/840                  ref|NZ_CP016411.1|      159156.0   159995.0   U12338     Resistance\n",
+      "GCF_001931595.1  tet(A)       tetracycline                                                     100.0              100.0              1200/1200                ref|NZ_CP016411.1|      172512.0   173711.0   AJ517790   Resistance\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Show all columns\n",
+    "column -s$'\\t' -t -n out/detailed_summary.tsv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Isolate ID       Gene         %Identity          HSP Length/Total Length  Contig                  Data Type\n",
+      "GCF_001478105.1  IncI1        100.0              142/142                  ref|NZ_JYVD01000056.1|  Plasmid\n",
+      "GCF_001478105.1  IncX1        100.0              373/373                  ref|NZ_JYVD01000049.1|  Plasmid\n",
+      "GCF_001478105.1  blaCMY-2     100.0              1146/1146                ref|NZ_JYVD01000056.1|  Resistance\n",
+      "GCF_001931595.1  None                                                                             Plasmid\n",
+      "GCF_001931595.1  aac(3)-IVa   99.87299999999999  786/786                  ref|NZ_CP016411.1|      Resistance\n",
+      "GCF_001931595.1  aph(3')-Ia   100.0              816/816                  ref|NZ_CP016411.1|      Resistance\n",
+      "GCF_001931595.1  aph(4)-Ia    100.0              1026/1026                ref|NZ_CP016411.1|      Resistance\n",
+      "GCF_001931595.1  blaCTX-M-65  100.0              876/876                  ref|NZ_CP016411.1|      Resistance\n",
+      "GCF_001931595.1  dfrA14       99.79299999999999  483/483                  ref|NZ_CP016411.1|      Resistance\n",
+      "GCF_001931595.1  floR         98.18799999999999  1214/1215                ref|NZ_CP016411.1|      Resistance\n",
+      "GCF_001931595.1  gyrA (D87Y)  99.431             2637/2637                ref|NZ_CP016410.1|      Resistance\n",
+      "GCF_001931595.1  sul1         100.0              840/840                  ref|NZ_CP016411.1|      Resistance\n",
+      "GCF_001931595.1  tet(A)       100.0              1200/1200                ref|NZ_CP016411.1|      Resistance\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Show a subset of columns\n",
+    "cut -f 1,2,4,6,7,11 out/detailed_summary.tsv | column -s$'\\t' -t -n"
```

Comment:
Could you maybe remove the column showing **HSP Length/Total Length** here? That way, when showing the notebook on Jupyter the results won't wrap to a new line.